lib/sync-settings.coffee

```diff
@@ -1,9 +1,8 @@
 # imports
 {BufferedProcess} = require 'atom'
-GitHubApi = require 'github'
-_ = require 'underscore-plus'
-PackageManager = require './package-manager'
 fs = require 'fs'
+_ = require 'underscore-plus'
+[GitHubApi, PackageManager] = []
```

Comment:
Will you add some explanation as comment to make it clear why the import is actually during package activation